assets/js/audioManager.js

```diff
@@ -0,0 +1,32 @@
+export default class AudioManager {
+  static AUDIO_PREF_KEY = "audioPreferences"
+
+  static effects = {
+    attack: new Audio("/audio/click.wav"),
+    win: new Audio("/audio/win.wav")
+  }
+
+  constructor({ storage = localStorage } = {}) {
+    this.storage = storage
+
+    if (this.storage.getItem(AudioManager.AUDIO_PREF_KEY) == null) {
+      this.storage.setItem(AudioManager.AUDIO_PREF_KEY, true)
+    }
+
+    window.addEventListener("phx:play-sound", event => {
```

Comment:
One detail we will have to keep in mind is that this object should be a singleton. I didn't want to add all that overhead since we only have one main view were everything happens from. But down the line we might have to do things differently.